src/views/Metrics/constants/chart-data.js

```diff
@@ -0,0 +1,43 @@
+import Translation from '@/plugins/I18n'
+
+// eslint-disable-next-line id-length
+const { t } = Translation.global
+
+export const RESET_COUNT = 0
+
+export const MIN_COUNT = 4
+
+export const MAX_COUNT = 6
+
+export const TO_FIXED_PERCENTAGE = 2
+
+export const TO_FIXED_DATA_VOLUME = 1
+
+export const SCREEN_SMALL_BREAKPOINT = 1024
+
+export const SCREEN_XSMALL_BREAKPOINT = 540
+
+export const MEAN_LINE_LABEL = t('chartContent.meanLineLabel')
```

Comment:
Copiar texto do arquivo de i18n
```suggestion
export const MEAN_LINE_LABEL = ""
```